tests/saw/sike_r1/proof/sidh.saw

```diff
@@ -29,6 +29,14 @@ let fp2_encode_spec = do {
     crucible_points_to encp (tm enc);
 };
 
+let sike_p503_r1_publickey_validation_spec = do {
+    (PKB, PKBp) <- A_f2elm_t "PKB";
```

Comment:
This is fine the way it is, but you could simplify this line (and similarly the next line) to `PKBb <- crucible_alloc_readonly f2elm_t;`. All you need is the pointer, and `A_f2elm_t` has some extra stuff that lets you say things about the value associated with that pointer (which you don't use in this spec).